server/routes/companies.js

```diff
@@ -2,14 +2,174 @@
 var express = require('express');
```

Comment:
Just as a general note, it seems like some of the indents are made of spaces and some are tabs. That doesn't affect much for JS implementation, but for languages like Python which are whitespace-dependent there can be issues. Also, developers will have different tab-to-space conversions (e.g. on my computer a tab can be the width of 2 spaces, while on Swetha's it may be 4 spaces).

TLDR: a non-issue for now, but something to keep in mind. I'm going to suggest using the 2 space width though since the default files use that.